contracts/src/compute_engines/rebase_denomination/IRebaseDenomination.cairo

```diff
@@ -0,0 +1,37 @@
+%lang starknet
+
+@contract_interface
+namespace IOracleController:
```

Comment:
Is this the name we want?

Reply:
No, good catch. Let's do `IOracleRebaser`?

Reply:
Okay. I think it's also called oracle controller for `IYieldCurve.cairo` fyi.

Reply:
Fixed in https://github.com/42labs/Empiric/pull/156/commits/9bad40bd3660516bdeea57646d2d9b60addceef5